app/grants/models.py

```diff
@@ -553,7 +567,8 @@ class Meta:
         null=True,
         blank=True,
     )
-    categories = models.ManyToManyField(GrantCategory, blank=True)
+    categories = models.ManyToManyField(GrantCategory, blank=True) # TODO: REMOVE
+    tags = models.ManyToManyField(GrantTag, blank=True)
```

Comment:
I think there is a confusion here tags doesn't relate to types, could be any combination of type / tag as the users will create those